src/libs/ajax/DataRepo.ts

```diff
@@ -2,17 +2,14 @@ import * as _ from 'lodash/fp';
 import {
   convertDatasetAccessRequest,
   convertDatasetParticipantCountRequest,
-  DatasetAccessRequest,
-  DatasetAccessRequestApi,
+  DatasetAccessRequest as DatasetAccessRequestUtils,
```

Comment:
This seems like the wrong name to me. It's true that this type is defined in a `Utils` file, but its meaning isn't `Utils` related. If it's the API type, then it should be defined in this file instead. Also there's no other `DatasetAccessRequest` in scope so no aliasing is needed in either case.

Reply:
I agree that type isn't `Utils` related, I wasn't ready with my PR so I should have left this as a draft. However, there is a `DatasetAccessRequest` in scope here https://github.com/DataBiosphere/terra-ui/blob/25f7a33363fb902fbbc4e3553e14cbbc1574e3bf/src/libs/ajax/DataRepo.ts#L129